openshift/tools/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.7
 
 toolchain go1.23.6
 
-require github.com/openshift/cluster-capi-operator/manifests-gen v0.0.0-20250326163512-302c8534aab9
+require github.com/openshift/cluster-capi-operator/manifests-gen v0.0.0-20250509150604-ac5aa33f14d7
```

Comment:
Hm, do we want to use the latest `release-4.19` branch of the manifests-gen?
This might work too, but keeping the same branch level reference might be better?

Reply:
If you think that's preferable I'll close this and create a replacement with the 4.19 tag, but the 4.19 branch of capi operator hasn't merged yet so that will have to wait

Reply:
I think it might be preferrable as an approach yeah. But if we are in a hurry we can merge this and create a new Pr once the capi operator one has merged.

Reply:
@JoelSpeed you can now use cluster-capi-operator's [72ad6eb](https://github.com/openshift/cluster-capi-operator/commit/72ad6eb01a1ffd8deb49e83f74caa7aed8ae1eff) commit, which landed in release-4.19 for this.